activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -576,7 +576,6 @@ def execute_and_clear(sql, name, binds, prepare: false)
             result = exec_cache(sql, name, binds)
           end
           ret = yield result
-          result.clear
```

Comment:
Should this be delegating the `clear` call?

Reply:
@sgrif I addressed that in the description.  I don't think the call is necessary.

Reply:
I updated the comment to be more explicit about my concerns but I was a bit too slow in editing it.

Reply:
@sgrif ah yes.  I think we should remove this method entirely.  I just left it here 1) to make the diff as small as possible, and 2) to explicitly call out the change to see if anyone knew if it's necessary.

If we're OK with removing the `clear` call, then I'll follow up and rm this method.